proof_of_stake/src/epoched.rs

```diff
@@ -128,13 +128,16 @@ pub enum DynEpochOffset {
     PipelineLen,
     /// Offset at unbonding length.
     UnbondingLen,
+    /// Offset at pipeline length - 1.
+    PipelineLenMinusOne,
```

Comment:
It's not a biggie, but maybe we could just call this case `One` and set it to `1`? Pipeline is always going to stay at `2`

Reply:
The only counterpoint to this would be in tests where we choose arbitrary PoS params - if in those tests we want to ultimately test txs to change the commission rate, then this should stay the same. But otherwise, I think this works too.

Reply:
ah, good point, let's keep it as is then!